### PR TITLE
Add Feature mobile skeleton

### PR DESCRIPTION
**Pull request by Wikitelmex**
- Add nav with flex
- Add section with about me
- Add image background
- Including Fonts
- Using Flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,39 @@
...
+    <link href='https://fonts.googleapis.com/css2?family=Poppins' rel='stylesheet'>
+
 </head>
 <body>
-    
+    <header>
+
+        <nav>
+            <ul>
+                <li class="my-logo">Alex Dev</li>
```
Comment (safafa) [inline index.html]: - [ ] The logo should be a clickable link. Kindly use an `<a>`  tag.
- [ ] As you can see here 
image
The logo and the menu button are aligned. 
image

Your header looks a bit different kindly fix the issue.

File: index.html [hunk 2]
```diff
@@ -5,8 +5,39 @@
...
+                </li>
+            </ul> 
+        </nav>
+
+    </header>
+    <section>
+        <h1>Hey There. I'm Alex</h1>
+        <h2>I'm a Software Developer</h2>
+        <p>I can help you build a product, feature or website Look trough some of my work and experience! If you like what you see and have a project you need coded, don't heastiate contact me.</p>
+    </section>
```
Comment (safafa) [inline index.html:46]: - [ ] As you can see in the original design. The section has a list of icons. Kindly add the missing icons. Following the original design is a must for this project.
image

- [ ] I think your version of the project looks a bit different from the original design in terms of margins positioning. Kindly follow the original design.
image

File: style/style.css [hunk 3]
```diff
@@ -1,3 +1,77 @@
...
+  line-height: 64px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: left;
+  color: #ff6b00;
+}
+
+section {
+  display: flex;
+
+  /* border: 5px solid red; */
```
Comment (safafa) [inline style/style.css]: - [ ] For better code readability always make sure to remove the commented code. Kindly remove all the commented code.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Wikitelmex , You did a great job so far 👏🏼 👏🏼 

- The repo looks professional.
- Commit messages are descriptive.
- The Pull Request is also descriptive.
- Good job on implementing the required sections.

However, here are some points that you need to work on to get your project approved.

## Required changes :

Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (Wikitelmex) [reply index.html]: - Logo now is a clickable link
- Logo and menu aligned
- List of Icons added
- Margins corrected of content
- Style improvements of code readability

**Review by karvel-code — APPROVED** [review]
Hi @Wikitelmex , 

## Status: Approved! 🎉🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**